js/src/forum/components/TokensManager.js

```diff
@@ -0,0 +1,62 @@
+import Component from '../../common/Component';
+import LoadingIndicator from '../../common/components/LoadingIndicator';
+import Button from '../../common/components/Button';
+import humanTime from '../../common/helpers/humanTime';
+
+export default class TokensManager extends Component {
+  init() {
+    this.refresh();
+  }
+
+  refresh() {
+    this.tokens = null;
+
+    app.request({
+      method: 'GET',
+      url: app.forum.attribute('apiUrl') + '/tokens',
+    }).then(payload => {
+      this.tokens = app.store.pushPayload(payload);
+      m.redraw();
+    });
+
+    // TODO: pagination
+  }
+
+  view() {
+    if (this.tokens === null) {
```

Comment:
Why not use a this.loading attribute?

Reply:
That's how I have usually been doing it for resource loading. null = not fetched, [] = no results, [...] = results.

I usually only use an additional loading variable when an action is being performed after clicking a button.

Reply:
Hmm it might make sense to standardize this across other views, super low priority tho